packages/react-layouts/src/Authentication/Authentication.md

```diff
@@ -0,0 +1,107 @@
+## Overview
+
+`Authentication` provides a standardized layout for authentication screens used for login, registration, "forgot password", or similar use-cases. It is designed to be composed with the `Form` component to provided whatever authentication-related use input is needed.
+
+# Example: Login
+
+```jsx
+import { Form } from '@puppet/react-components';
+
+<Authentication product="My Product" title="Sign in to My Product">
```

Comment:
I wonder if using a real `product` as an example or at least an indication of where to find "the official set" would help.